### PR TITLE
Elaborate on documentation for JetBrains releases

### PR DESCRIPTION
**Pull request by olafurpg**
## Test plan

Only changing  docs.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -116,7 +116,25 @@ After doing that:
 
 ## Publishing a New Release
 
-We plan to make releases every other Monday. Nightly version can be released as often as there is a need.
+
+```mermaid
+graph TD;
+    Title["JetBrains plugin release"] --> stable;
+    Title --> nightly;
+    stable --> push_stable["push git tag"];
```
Comment (pkukielka) [inline CONTRIBUTING.md:124]: Today we are using `./scripts/push-git-tag-for-next-release.sh` for that (it have some safeguards and also suggests you proper release branch name):

```
➜  jetbrains git:(pkukielka/move-session-lifecyccle-management-to-agent) ✗ ./scripts/push-git-tag-for-next-release.sh --patch
++ git symbolic-ref --short HEAD
+ CURRENT_BRANCH=pkukielka/move-session-lifecyccle-management-to-agent
+ '[' pkukielka/move-session-lifecyccle-management-to-agent '!=' main ']'
+ echo 'Warning: You are not on the '\''main'\'' branch. You are on '\''pkukielka/move-session-lifecyccle-management-to-agent'\''.'
Warning: You are not on the 'main' branch. You are on 'pkukielka/move-session-lifecyccle-management-to-agent'.
+ read -p 'Are you sure you want to proceed? (y/n): ' proceed
Are you sure you want to proceed? (y/n):
```

Comment (olafurpg) [reply CONTRIBUTING.md:124]: This is mentioned in the "push a git tag" section below the diagram.

Comment (pkukielka) [reply CONTRIBUTING.md:124]: Ah, it was not expanded in the diff and I missed it, thanks!

**Review by pkukielka — APPROVED** [review]
LGTM

**Pull request merged**